src/main/resources/reference.conf

```diff
@@ -411,6 +411,16 @@ cassandra-snapshot-store {
   # If the table doesn't exist it is automatically created.
   config-table = "config"
 
+  # Set this to on to only use Cassandra 2.x compatible features,
+  # i.e. if you are using a Cassandra 2.x server.
+  # To run tests with Cassandra 2.x server you have to do the following:
+  # - start Cassandra 2.x server on default port 9042, with empty data directory
```

Comment:
This has changed while this PR is in progress.

We now run the tests against docker rather than an embedded Cassandra per tests.  We did this under https://github.com/akka/akka-persistence-cassandra/pull/454 to speed up the build by 10 minutes.

So now all you need to do is change the docker-compose file to the desired cassandra version or start cassandra on port 9042 in some other way. (see https://github.com/akka/akka-persistence-cassandra/blob/master/CONTRIBUTING.md#running-the-tests)

Reply:
This PR is against the release-2.4 branch which doesn't have the docker files.

Reply:
:+1: Would be great to get this in master as well